src/Compilers/Core/MSBuildTaskTests/CopyRefAssemblyTests.cs

```diff
@@ -94,7 +94,13 @@ public void SourceNotAssemblyWithDestination()
             };
 
             Assert.True(task.Execute());
-            Assert.False(string.IsNullOrEmpty(engine.Log));
+
+            AssertEx.AssertEqualToleratingWhitespaceDifferences($$"""
+                Could not extract the MVID from "{{source.Path}}". Are you sure it is a reference assembly?
+                Copying reference assembly from "{{source.Path}}" to "{{dest}}".
+                """,
+                engine.Log);
+
             Assert.Equal("test", File.ReadAllText(dest.Path));
         }
```

Comment:
>} [](http://example.com/codeflow?start=8&length=1)

Can we test different MVIDs by creating files based on DLLs from `TestResources`? See `ReferenceManagerTests.ReferenceResolution1()` for instance.